### ISSUE TITLE
Bring some color back to new modmail

### ISSUE THREAD

**Issue by creesch** [toolbox-team/reddit-moderator-toolbox#402]
Specifically regarding the "replying as" dropdown. I am thinking something like this 

![image](https://user-images.githubusercontent.com/2502190/96754755-0bfb2380-13d2-11eb-9376-67823a0a15c7.png)

@SpyTec suggested we'd also color the submit button which might be a good idea. 

The only real challenge here is that there are no easy to use css classes so things have to be dynamically on page load and when clicking the dropdown.

**Comment by creesch** [comment]
Actually, all of it can be done with css looking into it further. 

```css
.FancySelect__option:nth-child(2) {
  color: #0079d3;
}
.FancySelect__option:nth-child(2):hover {
  color: #fff;
  background: #0079d3;
}

.FancySelect__option:nth-child(3) {
  color: #46d160;
}

.FancySelect__option:nth-child(3):hover {
  color: #fff;
  background: #46d160;
}
    
.ThreadViewerReplyForm__replyButton.m-internal {
  background: #46d160;
}

.m-internal .FancySelect__value {
  color: #46d160;
}
```

Looks to do nicely .

![image](https://user-images.githubusercontent.com/2502190/96756672-aa888400-13d4-11eb-8dc0-9437acc80293.png)

**Comment by Geo1088** [comment]
fixed in #406 for 5.5

**Comment by creesch** [comment]
Oh actually there is a bit of a bug with the current CSS, totally forgot I noticed that. So I am going to reopen it and fix it today.
